testing/src/main/java/org/oppia/android/testing/home/HomeFragmentTestHelper.kt

```diff
@@ -0,0 +1,5 @@
+package org.oppia.android.testing.home
+
+/** This helper allows tests to easily set various HomeFragment states. */
+class HomeFragmentTestHelper {
```

Comment:
Please fill this in, or remove if it isn't needed.

Reply:
Removed! Using StoryProgressTestHelper